src/mlpack/methods/ann/ffn_impl.hpp

```diff
@@ -52,6 +52,32 @@ FFN<OutputLayerType, InitializationRuleType, CustomLayers...>::~FFN()
       boost::apply_visitor(deleteVisitor));
 }
 
+template<typename OutputLayerType, typename InitializationRuleType,
+         typename... CustomLayers>
+void FFN<OutputLayerType, InitializationRuleType, CustomLayers...>::
+  CheckInputDim(size_t numRows, const char *functionName)
```

Comment:
```suggestion
    CheckInputDim(size_t numRows, const char *functionName)
```

The wrapped line should be indented twice. :+1: